pkg/extensions/search/schema.resolvers.go

```diff
@@ -579,6 +579,35 @@ func (r *queryResolver) BaseImageList(ctx context.Context, image string) ([]*gql
 	return imageList, nil
 }
 
+// Image is the resolver for the Image field.
+func (r *queryResolver) Image(ctx context.Context, image string) (*gql_generated.ImageSummary, error) {
+	layoutUtils := common.NewBaseOciLayoutUtils(r.storeController, r.log)
+	img, errImage := layoutUtils.GetImage(image)
```

Comment:
Why use this? Just call `GetImageDirAndTag` to obtain the repo name and tag.
There's no need to implement `layoutUtils.GetImage(image)`, you don't use any of that data.